apps/jenkins/jenkins/ptlsbox/jenkins-azure-vm-agent.yaml

```diff
@@ -31,71 +98,24 @@ spec:
                     resourceGroupReferenceType: "new"
                     maxVirtualMachinesLimit: 10
```

Comment:
do you need to bump this, if 10 agents were running on the first template then no arm ones would be able to run which would prevent builds if someone has requested only ARM?